src/unix/linux_like/linux/musl/lfs64.rs

```diff
@@ -0,0 +1,251 @@
+#[no_mangle]
```

Comment:
We shouldn't use `no_mangle` here: we don't actually want to define a `creat64` symbol, we just want what is effectively a `#define` alias for it.

Reply:
In fact these should be `#[inline]`.

Reply:
Hmmm, correct, I was under the impression that `extern "C"` functions are implicitly `#[no_mangle]` but it seems that's not the case.

`#[inline]` or `#[inline(always)]`?  This might be a valid use case for the rarely used second form...

Reply:
Please use plain `#[inline]`. The general policy is to only use `#[inline(always)]` for performance reasons if justified by benchmarks.

Reply:
No problem, done!